helpers/auth/allowed_requests.py

```diff
@@ -0,0 +1,13 @@
+import os
+from graphql import GraphQLError
+from flask import request
+
+
+def validate_origins():
+    "validate requests for production environments"
```

Comment:
consider placing the docstrings in `"""triple double quotes"""`

Reply:
noted